DTXMania/Code/Stage/04.Config/CActConfigList.cs

```diff
@@ -725,17 +735,17 @@ public void tSetupItemList_Drums()
             this.iDrumsHAZARD = new CItemToggle("HAZARD", CDTXMania.ConfigIni.bHAZARD,
                 "ドSハザードモード\n" +
                 "GREAT以下の判定でも回数が減ります。",
-                "Turn ON to decrease the damage gauge if the judgment\nis anthing less than PERFECT. Good Luck!",
+                "Turn ON to decrease the excite gauge if the judgment is anthing less than PERFECT.")
```

Comment:
hazard greatly increases the damage to the excite gauge (first introduced in v2) with no more than 2 poor/miss
(see [bemaniwikiv2](https://bemaniwiki.com/index.php?cmd=read&page=GuitarFreaksV2%A1%F5DrumManiaV2&word=hazard) and [this gameplay video (skip to 1:48) from DrumMania V3 MODEL FT2 Miracle Rank SS](https://www.dailymotion.com/video/xzcfu))

this should be reworded